documentation-website/Writerside/topics/Transactions.md

```diff
@@ -261,28 +261,28 @@ simply relayed to the database.
 **db:** This parameter is optional and is used to select the database where the transaction should be
 settled ([see the section above](#working-with-multiple-databases)).
 
-**Transaction Repetition Attempts**
+**Transaction Maximum Attempts**
 
-Transactions also provide a property, `repetitionAttempts`, which sets the number of retries that should be made if an
-SQLException occurs inside the `transaction` block. If this property is not set, any default value provided
-in `DatabaseConfig` will be used instead:
+Transactions also provide a property, `maxAttempts`, which sets the maximum number of attempts that should be made to perform a transaction block.
+If this value is set to 1 and an SQLException occurs inside the transaction block, the exception will throw without performing a retry.
```

Comment:
Not sure, but probably a mistake and should be `the exception will be thrown without performing a retry`